src/core.js

```diff
@@ -1005,7 +1005,7 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
 
     instance.runHooks('beforeInit');
 
-    if (isMobileBrowser()) {
+    if (isMobileBrowser() || isIpadOS()) {
```

Comment:
Can I ask you to add a test to mobiles that would cover that change? Test that checks if the root element of the table got the `mobile` class.

Reply:
Test added https://github.com/handsontable/handsontable/pull/7677/commits/39e9ed18d1ef4c8ac0c823b960217345c4bb6d38